From: Kyle McMartin <kyle@phobos.i.jkkm.org>
Date: Tue, 30 Mar 2010 00:04:29 -0400
Subject: [PATCH] die-floppy-die

Kill the floppy.ko pnp modalias. We were surviving just fine without
autoloading floppy drivers, tyvm.

--- a/drivers/block/floppy.c	2026-08-03 01:24:24.000000000 +0200
+++ b/drivers/block/floppy.c	2026-08-09 23:35:42.277464975 +0200
@@ -5016,7 +5016,7 @@ static const struct pnp_device_id floppy
 	{ }
 };
 
-MODULE_DEVICE_TABLE(pnp, floppy_pnpids);
+/* MODULE_DEVICE_TABLE(pnp, floppy_pnpids); */
 
 #else
 
